docs/dsp-tools-xmlupload.md

```diff
@@ -340,10 +335,10 @@ Attributes:
 - `permissions`: Permission ID (optional, but if omitted, users who are lower than a `ProjectAdmin` have no permissions at all, not even view rights)
 - `comment`: a comment for this specific value (optional)
 
-A property with two color values would be defined as follows:
+Example of a property with a public and a hidden color value:
 ```xml
 <color-prop name=":hasColor">
-    <color>#00ff66</color>
+    <color permissions="prop-default">#00ff66</color>
     <color>#ff00ff</color>
 </color-prop>
```

Comment:
I can understand your point,  but it is very important for me to prevent users from creating a hidden value (`<color>#ff00ff</color>`). Unfortunately, the docs have only examples of hidden values until now. This is dangerous. So I wanted to find a way how I can grab the user's attention, and teach him this important difference.

Reply:
On the long run, I could perhaps disallow the construct `<color>#ff00ff</color>`, and force the user to write `<color permissions="None">#ff00ff</color>`. This would prevent mistakes, but it would be a breaking change.